src/audio/eq_fir/eq_fir.c

```diff
@@ -723,40 +703,30 @@ static int eq_fir_copy(struct comp_dev *dev)
 {
 	struct comp_copy_limits cl;
 	struct comp_data *cd = comp_get_drvdata(dev);
-	struct fir_state_32x16 *fir = cd->fir;
-	int nch;
+	int n;
 
 	tracev_eq_with_ids(dev, "eq_fir_copy()");
 
 	/* Get source, sink, number of frames etc. to process. */
 	comp_get_copy_limits(dev, &cl);
 
-	nch = cl.source->channels;
-
-	/* Check if number of frames to process if it is odd. The
-	 * optimized FIR function to process even number of frames
-	 * is lower load than generic version. In that case process
-	 * one frame with generic FIR and the rest with even frames
-	 * number FIR version.
+	/* Process only even number of frames with the FIR function. The
```

Comment:
@singalsu typically multi-line commens should start on the line after the first "/*". I know this has been there from before but maybe better to fix it while you're amending the comment?

Reply:
OK, I'll change. I've been doing this style comments always. Didn't know...

Reply:
I've now changed this. There was also one comment before this with wrong style.